rasr/config.py

```diff
@@ -203,13 +219,17 @@ def __repr__(self):
             max_len = max(map(lambda t: len(t[1]), g))
             for t in g:
                 buf.append("%-*s = %s" % (max_len, t[1], t[2]))
-
+        buf.append(self._epilog)
         return "\n".join(buf)
 
     def __sis_state__(self):
         result = {"tree": self.__dict, "value": self._value}
         return result
 
+    def _sis_hash(self):
```

Comment:
Will this not ignore the regular contents of the config? Or is it complementary to the `__sis_state__` ?

Also, to preserve the hashes of older configs, if the value is `""` it should be excluded from the hash.